types/types.go

```diff
@@ -0,0 +1,6 @@
+package types
+
+// GenerateURLRequest is a struct that represents request body of /generate-presigned-url endpoint
+type GenerateURLRequest struct {
+	FileName string `json:"file"`
```

Comment:
can we replace the file with fileName

Reply:
Done via [`e6750bd`](https://github.com/ZeStream/zestream-server/pull/4/commits/e6750bd7f2fb93959885cf58aed6a7250973d8f5)